permutation/population_permutation.Rmd

```diff
@@ -80,10 +80,10 @@ Leave voters into another array, called `leave_ages`.
 # Extract Remain and Leave voters into their separate own data frames.
 # As above, we are just using Pandas here to get arrays for the
 # Remain and Leave voter ages.
-remainers = brexit_df[brexit_df['brexit_vote'] == 'Remain']
+remainders = brexit_df[brexit_df['brexit_vote'] == 'Remain']
```

Comment:
but we should also ignore these `remainers` probably in all of their forms right?

Reply:
I skipped them too now!